config/crd/bases/infrastructure.cluster.x-k8s.io_oscclusters.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.15.0
+    controller-gen.kubebuilder.io/version: v0.14.0
```

Comment:
why did you dowgrade this version ?

Reply:
Didn't do it on purpose, I am not up to date on my controller-gen version, will change that.